common/types/head_tracker.go

```diff
@@ -24,6 +24,10 @@ type HeadTracker[H Head[BLOCK_HASH], BLOCK_HASH Hashable] interface {
 //
 //go:generate mockery --quiet --name HeadTrackable --output ./mocks/ --case=underscore
 type HeadTrackable[H Head[BLOCK_HASH], BLOCK_HASH Hashable] interface {
+	// OnNewLongestChain sends a new head when it becomes available. Subscribers can recursively trace the parent
+	// of the head to EVM_FINALITY_DEPTH blocks back. If this is not possible (e.g. due to recent boot, backfill not complete
```

Comment:
Can we update this env var to the toml config path? Or be even less specific?

Reply:
done